packages/frontend-2/lib/settings/graphql/queries.ts

```diff
@@ -3,12 +3,15 @@ import { graphql } from '~~/lib/common/generated/gql'
 export const settingsSidebarWorkspacesQuery = graphql(`
   query SettingsSidebarWorkspaces {
```

Comment:
No big deal, but I'd remove the `Workspaces` part from the query name - sure, currently its about workspaces, but ultimately its about pulling in all of the data that the SettingsSidebar needs, which may include other stuff in the future.